CMakeLists.txt

```diff
@@ -202,6 +202,22 @@ endif(ENABLE_ORC)
 ########################################################################
 add_subdirectory(docs)
 
+########################################################################
+# Setup Coverage
+########################################################################
+if(${CMAKE_BUILD_TYPE} STREQUAL "Coverage")
```

Comment:
With the CodeCoverage.cmake setting the Build_Type to anything other than Debug results in a warning. Either removing that check from there or leaving the `CMAKE_BUILD_TYPE` alone and just adding a switch to enable coverage would be an option.

Reply:
this is honestly just a mirror of what GR does; looking back, yes, having extra flags instead of a build type of its own would have been a worthwhile option, but I foreseeable won't have time to implement that.